core/src/main/resources/org/apache/spark/ui/static/webui.css

```diff
@@ -194,6 +194,24 @@ span.additional-metric-title {
   color: #777;
 }
 
+#additional-metrics {
+  float: left;
+  padding-bottom: 10px;
+}
+
+#links {
+  float: left;
+  padding-left: 100px;
+}
+
+#links a {
+  padding-left: 10px;
+}
+
+.clear {
```

Comment:
no, it's added to the "Summary Metrics …" <h4> element [here](https://github.com/apache/spark/pull/5121/files#diff-fa4cfb2cce1b925f55f41f2dfa8c8501R450)